src/prototypes/case-study-listings/example/example.twig

```diff
@@ -112,7 +112,7 @@
     </div>
   </div>
 
-  <div class="wrapper book-wrapper">
```

Comment:
I think there's a chunk of CSS that can be removed, too?

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1458/commits/ee9a7130233006ed3778d7b01723bb697ccb0cfb